### DISCUSSION TITLE
Next newsletter edition: How to secure your code with AI

### DISCUSSION THREAD

**Discussion by ettaboyle** [community/community#77489] [Announcements]
### Is this you? :flushed: :point_down:

<img width="360" alt="shift left" src="https://github.com/community/community/assets/75342807/432479ab-ab4c-4140-9e18-2592620ef10c">

No matter where you are in your security journey, this upcoming newsletter should help. We’ll explore how AI is redefining the developer experience by empowering you to fix issues in real-time time as you code—finally helping you achieve the true promise of “shift left”.

## ➡️  **[Subscribe now](https://resources.github.com/newsletter/)**

**Comment by user8104988254885** [comment]
![29f135d011628c5724df51a92c3713cd_0](https://github.com/community/community/assets/136828501/44098724-94b6-4048-ac9a-0f4c9c3ea3b6)

**Comment by itsbeelaw** [comment]
Needed! Thanks for this.

**Reply by Auwal-Ac** [reply]
Please I need help with money please my account number is 0005780086 jaiz bank

**Comment by Abdullah-s1** [comment]
> ## Next newsletter edition: How to secure your code with AI
> 
> ### Is this you? :flushed: :point_down:
> 
> 
> 
> <img width="360" alt="shift left" src="https://github.com/community/community/assets/75342807/432479ab-ab4c-4140-9e18-2592620ef10c">
> 
> 
> 
> No matter where you are in your security journey, this upcoming newsletter should help. We’ll explore how AI is redefining the developer experience by empowering you to fix issues in real-time time as you code—finally helping you achieve the true promise of “shift left”.
> 
> 
> 
> #### Join us for tips and tricks on how you can use AI to block vulnerabilities in the IDE and easily secure your code today.
> 
> ## ➡️  **[Subscribe now](https://resources.github.com/newsletter/)**

**Comment by EngNatheerAlnaqep** [comment]
how can i add a txt record  dns in my website asp.net mvc

**Reply by williamfredeveret** [reply]
Fttyuui

**Reply by CLFIELDS** [reply]
I'm not sure what to expect

**Reply by tatianak22608** [reply]
[]()

**Comment by josephmulenga493** [comment]
I trade on aave flashloan app but did not pay why

**Comment by Nancyberry31** [comment]
![IMG_20231224_033124_773](https://github.com/community/community/assets/154547395/07190ed0-3c2f-4c3e-a013-1314ead0f29c)
Nancy Berry

**Comment by Yu7059** [comment]
KaliLinux

**Comment by MamoNew** [comment]
KaliLinux

**Comment by israk-hossain** [comment]
using linux

**Comment by israk-hossain** [comment]
using kalilinux

**Comment by Auwal-Ac** [comment]
Ok me i am auwal ac money I need because I help some people so the account
is 0005780086 jaiz bank

**Comment by Kmi2024** [comment]
Alguém sabe sobre o projeto Io433

**Comment by novium2** [comment]
Retiring this post as it is over 6 months old. If you have any questions, consider starting a [new post](https://github.com/orgs/community/discussions/new/choose).
